### DISCUSSION TITLE
Considering using Microsoft Edge WebView2 as the runtime on Windows?

### DISCUSSION THREAD

**Discussion by zhangz1han** [KevinnZou/compose-webview-multiplatform#199] [Ideas]
当前方案在 Windows 上需要下载庞大的运行时，文件高达 400+MB。而且受限于内地的网络环境，下载非常缓慢，而且容易失败

微软在 Windows 上提供了 Microsoft Edge WebView2 作为通用 WebView，不再需要每个应用重复下载。使用此运行时也许能提供更好的体验？

https://developer.microsoft.com/en-us/microsoft-edge/webview2

**Comment by Shabinder** [comment]
Related
https://github.com/KevinnZou/compose-webview-multiplatform/issues/256
